### DISCUSSION TITLE
InstantDB

### DISCUSSION THREAD

**Discussion by j3hn** [coollabsio/coolify#3499] [Service Template Requests]
https://www.instantdb.com/
The realtime client-side database

Now opensource
https://news.ycombinator.com/item?id=41322281

Self Hostable
https://github.com/instantdb/instant/tree/main/server

**Comment by kualta** [comment]
yes please!

**Comment by rusakovic** [comment]
Would be great to have . Seemps dokploy doesn't support s3 https://templates.dokploy.com/?q=Instant
